### PR TITLE
README: add repology badge

### PR DESCRIPTION
**Pull request by novakne**
Just add a link to repology.org in the README.md, I think it's all we need

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # river
 
+[![Packaging status](https://repology.org/badge/vertical-allrepos/river.svg)](https://repology.org/project/river/versions)
```
Comment (YerinAlexey) [inline README.md:3]: Maybe not as the first thing a newcomer sees? A dedicated "packaging status" section will be better IMO

Comment (novakne) [reply README.md:3]: I feel like I always saw this first in repos using this, but I let @ifreund decide if/where he wants it

Comment (novakne) [reply README.md:3]: I guess we could move it to the right side, see this [branch](https://github.com/novakne/river/tree/repology-badge-side)

Comment (YerinAlexey) [reply README.md:3]: > I guess we could move it to the right side, see this [branch](https://github.com/novakne/river/tree/repology-badge-side)

That looks good to me, and doesn't waste as much space

Comment (branwright1) [reply README.md:3]: Mayby better add it next to building? 
image

Comment (ifreund) [reply README.md:3]: Having this aligned to the right next to building looks good to me.

Comment (novakne) [reply README.md]: Done!

**Review by ifreund — APPROVED** [review]
Thanks!

**Pull request merged**